lib/matplotlib/colorbar.py

```diff
@@ -141,12 +141,12 @@
 Parameters
 ----------
 mappable
-    The :class:`~matplotlib.image.Image`,
-    :class:`~matplotlib.contour.ContourSet`, etc. to
-    which the colorbar applies; this argument is mandatory for the Figure
-    :meth:`~matplotlib.figure.Figure.colorbar` method but optional for the
-    pyplot :func:`~matplotlib.pyplot.colorbar` function, which sets the
-    default to the current image.
+    The `~matplotlib.cm.ScalarMappable` (i.e., `~matplotlib.image.Image`,
+    `~matplotlib.contour.ContourSet`, etc.) to which the colorbar
+    applies.  This argument is mandatory for the Figure method
+    `~matplotlib.figure.Figure.colorbar` but optional for the pyplot function
```

Comment:
Could use `.Figure.colorbar` and `.pyplot.colorbar`.

Reply:
done